concourse/tasks/publish-time-since.yaml

```diff
@@ -0,0 +1,22 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: gcr.io/gcp-guest/concourse-metrics
+    tag: latest
+
+inputs:
+- name: credentials
+
+params:
+  GOOGLE_APPLICATION_CREDENTIALS: "credentials/credentials.json"
+
+run:
+  path: /publish-time-since
+  args:
+  - -project-id=gcp-guest
+  - -metric-path=concourse/job/duration
```

Comment:
This may be a copy pasta error -> this metric should be different than the job duration metric. Maybe 'concourse/package/stable_stale_time' to follow the [naming conventions](https://cloud.google.com/monitoring/api/v3/naming-conventions)

Reply:
thanks for catching that, updated